testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -485,4 +480,72 @@ private void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
             onDelta().executeQuery("DROP TABLE default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "testDmlOperationsConfigDataProvider")
```

Comment:
This seems not to work on `DELTA_LAKE_OSS` 

```
 io.trino.tests.product.deltalake.TestDeltaLakeColumnMappingMode.testDmlOperationsWithColumnMapping [id, 5, true] (Groups: profile_specific_tests, delta-lake-exclude-91, delta-lake-databricks, delta-lake-oss, delta-lake-exclude-73) took 0.6 seconds
2023-01-30T16:25:46.4502308Z tests               | 2023-01-30 22:10:46 SEVERE: Failure cause:
2023-01-30T16:25:46.4503125Z tests               | io.trino.tempto.query.QueryExecutionException: java.sql.SQLException: org.apache.hive.service.cli.HiveSQLException: Error running query: org.apache.spark.sql.delta.ColumnMappingUnsupportedException: The column mapping mode `id` is not supported for this Delta version. Please upgrade if you want to use this mode.
2023-01-30T16:25:46.4504069Z tests               | 	at org.apache.spark.sql.hive.thriftserver.HiveThriftServerErrors$.runningQueryError(HiveThriftServerErrors.scala:44)
2023-01-30T16:25:46.4504897Z tests               | 	at 
```

Please investigate whether this is the case and if so pls remove `DELTA_LAKE_OSS` from the test `groups`

Reply:
It seems that `delta.columnMapping.mode` `id` is avaiable only on Delta `2.2.0` - we're currently using Delta `2.1.0`

See:
-  https://github.com/delta-io/delta/blob/v2.2.0/core/src/main/scala/org/apache/spark/sql/delta/DeltaColumnMapping.scala#L62-L63
- https://github.com/trinodb/docker-images/blob/master/testing/spark3-delta/Dockerfile#L18

I'll create a PR to upgrade Trino's `spark3-delta` image.

Reply:
Created https://github.com/trinodb/docker-images/pull/160

Reply:
update version trino's PR: https://github.com/trinodb/trino/pull/15914